relay/kusama/src/lib.rs

```diff
@@ -1859,6 +1867,11 @@ pub mod migrations {
 		parachains_inclusion::migration::MigrateToV1<Runtime>,
 		parachains_on_demand::migration::MigrateV0ToV1<Runtime>,
 		restore_corrupted_ledgers::Migrate<Runtime>,
+		// Migrate NominationPools to `DelegateStake` adapter. This is an unversioned upgrade.
+		pallet_nomination_pools::migration::unversioned::DelegationStakeMigration<
```

Comment:
Fine if this is called multiple times?

Reply:
Yeah it ignores pool if already migrated.